ONSdigital/census-rm-case-api-2021
# Create uacqid endpoint

# Motivation and Context
A new end-point is required that will generate a UAC QID Pair.

# How to test?
Check out census-rm-acceptance-tests, census-rm-docker-dev on branch create-uacqid-endpoint
Run mvn clean install on census-rm-case-api and start docker-dev
Run acceptance tests

# Links
https://trello.com/c/0Jmh2jK2/1003-rmc-134b-synchronous-uac-qid-creation-via-case-api-part-1-5
https://github.com/ONSdigital/census-rm-docker-dev/pull/23
https://github.com/ONSdigital/census-rm-kubernetes/pull/89
https://github.com/ONSdigital/census-rm-acceptance-tests/pull/97
